src/game.rs

```diff
@@ -66,10 +66,8 @@ impl Game {
     }
 
     /// Iterates over the live cells and calls the given closure with the position of each cell.
```

Comment:
I believe this comment is out of date now.

Reply:
Truee, hadn't noticed that thanks!

Reply:
Done!